CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.2.0
+
+* **chore**: Upgraded Android SHA request to new apis
```

Comment:
Can you change this to `Updated to use new APIs for requesting Android SHA`?

Reply:
Ofcourse!